README.md

```diff
@@ -1,2 +1,14 @@
 # join-fiware-lab-guide
-In this repository you can find the up-to-date guide on how your own OpenStack data center can Join FIWARE Lab, a community cloud offering free resources to FIWARE developers.
+
+In this repository you can find the up-to-date guide on how your own OpenStack
+data center can Join FIWARE Lab, a community cloud offering free resources to
+FIWARE developers.
+
+ReadTheDocs documentation is generated automatically out of [MarkDown (MD)](https://guides.github.com/features/mastering-markdown/)
```

Comment:
This process is a bit tricky, but I've done it once (for the recipes IIRC). Let me know if I could be of help.